src/System.Diagnostics.TraceSource/tests/TraceClassTests.cs

```diff
@@ -341,9 +351,42 @@ public void TraceTest01()
         }
 
         [Fact]
```

Comment:
I guess the difference is that FrameworkDescription includes a version number and TargetFrameworkName doesn't?